src/Libraries/Microsoft.Extensions.AI.Evaluation.NLP/SimpleWordTokenizer.cs

```diff
@@ -0,0 +1,210 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Text;
+using Microsoft.Shared.Diagnostics;
+
+namespace Microsoft.Extensions.AI.Evaluation.NLP;
+
+/// <summary>
+/// Tokenizes a string into segments using the common rules established by the NLTK word tokenizer.
+/// </summary>
+[Experimental("AIEVAL001")]
```

Comment:
As we discussed offline, let's make this `internal` for now. We can wait for feedback to see if we need extensibility for word tokenization.

Please remove the `[Experimental]` here and the `<InjectExperimentalAttributeOnLegacy>` from the project file as part of making this `internal`.

Reply:
I already pushed the change to make it internal. We are using experimental on other classes, so the injection should remain.